raiden/settings.py

```diff
@@ -47,3 +47,5 @@
 
 RED_EYES_CONTRACT_VERSION = '0.4.0'
 DEVELOPMENT_CONTRACT_VERSION = '0.3._'
+
+DEFAULT_MS_BROADCASTING_ROOM = 'monitoring_services'
```

Comment:
Is this supposed to be configurable by the user? if yes, is there a reason to allow this? if not, this should be in `constants.py`

Reply:
Right, fixed this.